eo-runtime/src/test/java/EOorg/EOeolang/EOio/EOstdinTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package EOorg.EOeolang.EOio;
+
+import java.io.InputStream;
+import java.io.ByteArrayInputStream;
+import org.eolang.Dataized;
+import org.eolang.PhCopy;
+import org.eolang.PhMethod;
+import org.eolang.Phi;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+
+/**
+ * Test case for {@link EOstdin}.
+ *
+ * @since 0.23
+ */
+public final class EOstdinTest {
+
+    private static final InputStream DEFAULT_STDIN = System.in;
+
+    @AfterAll
+    public static void restoreSystemInput() {
+        System.setIn(DEFAULT_STDIN);
+    }
+
+    @Test
+    public void nextLineOneLineTest() {
+        String expected = "this is a test input!";
+        mockSystemIn(expected + "\n");
+
```

Comment:
@Graur read this: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html